jobs/notify_admin/scheduler.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package notify_admin
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/mattermost/mattermost-server/v6/jobs"
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+)
+
+const schedFreq = 1 * time.Minute
```

Comment:
Is this intended to be 1 minute?

Reply:
Yes, it's a temp change to make testing on the spinwicks much faster. There's a do not merge label to remind me to correct this to the right time